frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -451,7 +451,7 @@ describe("Where's My Gene", () => {
 
       // Click into a cell type that has marker genes
       await page
-        .getByTestId("cell-type-info-button-adipose tissue-phagocyte")
+        .getByTestId("cell-type-info-button-adipose tissue-contractile cell")
```

Comment:
phagocytes are no longer in adipose tissue in the latest version of the snapshot. switching to contractile cell.

this is not related to any of the rollup changes since none of the existing pipeline code was changed. a new pipeline step was simply added.